packages/code-studio/index.html

```diff
@@ -16,6 +16,8 @@
     <link rel="manifest" href="/manifest.json" />
     <link rel="icon" href="%VITE_FAVICON%" />
     <title>Deephaven</title>
+    <link rel="stylesheet" href="/default_theme.css" />
+    <link rel="stylesheet" href="/custom_theme.css" />
```

Comment:
Is this more of a POC or intended implementation of how custom themes would be loaded?

It's probably preferable that they are split into separate files, and then loaded dynamically, such that theme files can be created and shared individually.

Reply:
I imagine the default themes would actually live in the codebase not in public. Those could be split up however.

For the custom ones, it's easiest to just have 1 file and statically load it. I could see the benefit of splitting them up, would just have to see how hard it is to determine the full list before our app code has actually loaded